README.md

```diff
@@ -1,2 +1,154 @@
-# code-engine-github-action
-Repository to demonstrate how Github Actions can be used to deploy your source code to IBM Cloud Code Engine
+# Code Engine GitHub Action
+
+This GitHub Action allows you to interact with IBM Cloud Code Engine. Deploy Apps, Jobs, and Functions. It offers flexibility for different deployment types and provides various configuration options.
+
+## Inputs
+
+| Name            | Required | Default Value |Description |
+|-----------------|----------|---------------|----------------------------------------------------------------|
+| `api-key`       | ✅      | -             | IAM API Key used to log into the IBM Cloud. Please store your IBM Cloud API key securely in your GitHub repository Secrets.|
+| `resource-group`| ❌       | Default       | An IBM Cloud Resource Group, a logical container for organizing and managing related cloud resources.|
+| `region`        | ✅      | -             | The geographical area where your Code Engine project is located.|
+| `project`       | ✅      | -             | A Code Engine Project, grouping your Apps, Functions, and Jobs.|
+| `entity`        | ✅      | -             | The type of entity to deploy (App, Function, Job). |
+| `name`          | ✅      | -             | The name of the App, Function, or Job.|
+| `runtime`       | ❌ | -             | The runtime used for the Function. Currently supported `nodejs-18` and `python-3.11` see [IBM Code Engine Function Runtimes](https://cloud.ibm.com/docs/codeengine?topic=codeengine-fun-runtime) for more information.|
+| `build-source`  | ❌       | .             | Path to the directory containing the source code.|
+| `cpu`           | ❌       | -             | CPU value set for your entity [Config for Functions](https://cloud.ibm.com/docs/codeengine?topic=codeengine-fun-runtime), [Codeengine Memory CPU combo](https://cloud.ibm.com/docs/codeengine?topic=codeengine-mem-cpu-combo)|
```

Comment:
link must point to cpu mem values that are allowed: https://cloud.ibm.com/docs/codeengine?topic=codeengine-mem-cpu-combo

Reply:
Already in the description